packages/connect-query/src/use-infinite-query.test.ts

```diff
@@ -375,6 +375,7 @@ describe("useSuspenseInfiniteQuery", () => {
     expect(result.current.isFetching).toBeFalsy();
   });
 
+  // eslint-disable-next-line vitest/expect-expect -- We are asserting via @ts-expect-error
```

Comment:
Turns out https://github.com/connectrpc/connect-query-es/pull/428 has type errors then went unnoticed because of this issue.

CI should definitely catch type errors in tests. WDYT, @paul-sachs

Reply:
Yeah, i'll take a look and see what needs to be fixed here.